charts/gardener/gardenlet/values.yaml

```diff
@@ -38,11 +38,17 @@ global:
       # kubeconfigSecret:
```

Comment:
Can you add a comment here, stating that this is the secret that the generated kubeconfig from TLS bootstrapping will be stored in, please?
Could be something along the lines from the Config API doc: https://github.com/gardener/gardener/blob/5b69a02f511af5d085da8976f3aa5345c354c44d/pkg/gardenlet/apis/config/v1alpha1/types.go#L94-L96

Reply:
Done, thanks!